docs/components/concepts/process-instance-creation.md

```diff
@@ -188,6 +188,59 @@ For each new message a new instance is created.
 
 A process can also have one or more [timer start events](/components/modeler/bpmn/timer-events/timer-events.md#timer-start-events). An instance of the process is created when the associated timer is triggered. Timers can also trigger periodically.
 
+## Tags (8.8+)
+
+Process instance tags are lightweight, immutable labels you can attach when creating a process instance via the v2 API or SDK. They help downstream workers and external systems make quick routing or decision choices without inspecting full variable payloads.
```

Comment:
🔧 I think we don't call it "v2 API" in the docs 😅 We could add a link to the endpoint?